nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h

```diff
@@ -248,6 +249,13 @@ struct nrf_wifi_fmac_callbk_fns {
 	void (*process_rssi_from_rx)(void *os_vif_ctx,
 				     signed short signal);
 #endif /* CONFIG_NRF700X_STA_MODE */
+#ifndef HOST_CFG80211_SUPPORT
```

Comment:
This name is confusing as cfg80211 is always in host and this actually means that there is no cfg80211 in host, we should use `CONFIG_NRF_VANILLA_NL` i.e., vanilla netlink compared to cfg80211 netlink.